### PR TITLE
pinned message event detail

### PR DESCRIPTION
**Pull request by sumanac**
Pinned unpinned event messages

### CODE REVIEW

**Comment by sumanac** [comment]
Will add change log

File: api-reference/beta/resources/messageunpinnedeventmessagedetail.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "messageUnpinnedEventMessageDetail resource type"
+description: "Represents the details of an event message about unpin message."
+author: "sumanac"
```
Comment (RamjotSingh) [inline api-reference/beta/resources/messageunpinnedeventmessagedetail.md:4]: If you are using your name as author remember to monitor for issues or you can put my name.

**Review by RamjotSingh — APPROVED** [review]

File: api-reference/beta/resources/messageunpinnedeventmessagedetail.md (hunk 1 above)
Comment (sumanac) [reply api-reference/beta/resources/messageunpinnedeventmessagedetail.md:4]: I can monitor. I will follow-up with you offline, how to set that up.

File: api-reference/beta/resources/messagepinnedeventmessagedetail.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+doc_type: resourcePageType
+---
+
+# messagePinnedEventMessageDetail resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the details of an event message about pin message. This message is generated when a message is pinned.
```
Comment (JarbasHorst) [inline api-reference/beta/resources/messagepinnedeventmessagedetail.md:16]: For discoverability and consistency, please add a reference link to chat. 

Please, apply this change also here:
- api-reference/beta/resources/messageunpinnedeventmessagedetail.md

File: changelog/Microsoft.Teams.Core.json [hunk 3]
```diff
@@ -5057,6 +5057,36 @@
...
+              "Description": "Added the [messageUnpinnedEventMessageDetail](https://docs.microsoft.com/en-us/graph/api/resources/messageUnpinnedEventMessageDetail?view=graph-rest-beta) resource type.",
+              "Target": "messageUnpinnedEventMessageDetail"
+            }
+          ],
+          "Id": "2a440a33-2393-43a7-b47e-db1d3acd7585",
+          "Cloud": "Prod",
+          "Version": "beta",
+          "CreatedDateTime": "2022-05-26T00:54:29.7304356Z",
+          "WorkloadArea": "Teamwork",
+          "SubArea": "Microsoft.Teams.Shifts"
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.Core.json:5087]: For event messages, we don't need to enter Shifts as a "SubArea" here. The value can be just an empty string => "SubArea": ""

**Comment by JarbasHorst** [comment]
Hello @sumanac, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [17118](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17118) into your branch to include my fixes and before addressing my comments.

File: api-reference/beta/resources/messagepinnedeventmessagedetail.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  "eventDateTime": "String (timestamp)",
+  "initiator": {
+    "@odata.type": "microsoft.graph.identitySet"
+  }
+}
+```
+
+
+## See also
+- For more information about other types of events, see [System messages](/graph/system-messages).
```
[the remark this answers is not in the document]
Comment (sumanac) [reply api-reference/beta/resources/messagepinnedeventmessagedetail.md:47]: Please check if this change is applied

File: concepts/system-messages.md [hunk 5]
```diff
@@ -61,6 +61,8 @@ For details, see [Change notifications for messages](teams-changenotifications-c
 | Members deleted | [membersDeletedEventMessageDetail](#members-deleted) | Chat, Channel, Team |
 | Members joined | [membersJoinedEventMessageDetail](#members-joined) | Chat |
 | Members left | [membersLeftEventMessageDetail](#members-left) | Chat |
+| Pin message | [messagePinnedEventMessageDetail](#pin-message) | Chat, Channel |
```
Comment (JarbasHorst) [inline concepts/system-messages.md:64]: The correct events for "Pin message" and "Unpin message" should be:
- Chat message pinned
- Chat message unpinned

Please, update lines 64 and 65 to contain the correct information. After that, please reorganize this table in alphabetical order, update all reference links (for example, #unpin-message should be "#chat-message-unpinned), and reposition the examples so they respect the current alphabetical order.

**Comment by JarbasHorst** [comment]
Hi @sumanac, thanks for addressing my previous comments. I left one additional. Please, take a look when you will have time.

The PR can be merged after we finished the content review work and the PR author set the label ready to merge. Currently, the content review work is still in progress. Please contact us directly (for example, via Teams) if you want to share some urgency about this PR.

Also, to avoid merge conflicts, please merge PR [17177](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17177) into your branch to include my fixes and before addressing my new comment.

File: concepts/system-messages.md (hunk 5 above)
Comment (sumanac) [reply concepts/system-messages.md:64]: It's not only chat message pinned. It applies to channels as well. You can also see the event name is "messagePinnedEventMessageDetail" Pin a message is a correct event in my opinion.

Comment (JarbasHorst) [reply concepts/system-messages.md:64]: In the "Supported system message events" table, all events listed describe an action that has already happened and the entries don't start with the verb, for example, Team archived. For consistency, I think that these entries should be updated. Now that you provided more details, I think that "Message pinned" and "Message unpinned" would be the correct names. I will take this topic for internal discussion with Laura and get back to you.

**Comment by JarbasHorst** [comment]
Hi @sumanac, thanks for your feedback and for providing more details. I left additional comments. Please, take a look when you will have time.

File: api-reference/beta/resources/messagepinnedeventmessagedetail.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+doc_type: resourcePageType
+---
+
+# messagePinnedEventMessageDetail resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the details of an event message about a pinned [chatMessage](../resources/chatmessage.md). This message is generated when a chat message is pinned.
```
[the remark this answers is not in the document]
Comment (sumanac) [reply api-reference/beta/resources/messagepinnedeventmessagedetail.md:16]: I checked the channel pinned message and system messages are not showing. I removed the "Channel" from system message as well.

File: api-reference/beta/resources/messageunpinnedeventmessagedetail.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+doc_type: resourcePageType
+---
+
+# messageUnpinnedEventMessageDetail resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents the details of an event message about an unpinned [chatMessage](../resources/chatmessage.md). This message is generated when a chat message is unpinned.
```
[the remark this answers is not in the document]
Comment (sumanac) [reply api-reference/beta/resources/messageunpinnedeventmessagedetail.md:16]: same as pinned msg comment.

**Comment by sumanac** [comment]
@JarbasHorst - I only found 2 comments and I resolved it with justification. Please merge this PR.

**Comment by sumanac** [comment]
Please merge this PR. This is open for really long. Please feel free to ping me if there is additional comments, so I can take care of that ASAP.

**Comment by JarbasHorst** [comment]
@sumanac, thanks for addressing my previous comment.

@Lauragra this PR is ready for final review. Please, merge PR [17226](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17226) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**